.azure/azure-linux-template.yml

```diff
@@ -45,8 +45,11 @@ jobs:
           export PATH=$HOME/.local/bin:/opt/ghc/$GHCVER/bin:/opt/happy/1.19.5/bin:/opt/alex/3.1.7/bin:$PATH
           ;;
       esac
-      if ! [ "$BUILD" = style ]; then echo "$(ghc --version) [$(ghc --print-project-git-commit-id 2> /dev/null || echo '?')]"; fi
-         set -ex
+      if [ "$BUILD" = stack ];
+      then echo "$(stack ghc -- --version)";
+      else echo "$(ghc --version) [$(ghc --print-project-git-commit-id 2> /dev/null || echo '?')]";
+      fi
+      set -ex
```

Comment:
There's a semantic change here. Previously the second `echo` command would be run if the first one failed and only in that case.

Now, the second `echo` command is run whenever `$BUILD$` is not `stack`